tsdb/config_test.go

```diff
@@ -9,12 +9,40 @@ import (
 
 func TestConfig_Parse(t *testing.T) {
 	// Parse configuration.
-	var c tsdb.Config
+	c := tsdb.NewConfig()
 	if _, err := toml.Decode(`
-enabled = false
+dir = "/var/lib/influxdb/data"
+wal-dir = "/var/lib/influxdb/wal"
 `, &c); err != nil {
 		t.Fatal(err)
 	}
 
-	// TODO: add remaining config tests
+	if err := c.Validate(); err != nil {
+		t.Errorf("unexpected validate error: %s", err)
+	}
+
+	if c.Dir != "/var/lib/influxdb/data" {
```

Comment:
Done. The order is still weird because we don't have a standard for the order of got and expected (I wanted it to be the same as other sections of the code), but we'll have a conversation about that soon and I'll personally go through the test files to switch the order when we do.